oxide-auth/src/endpoint/access_token.rs

```diff
@@ -225,14 +242,15 @@ impl<'a, R: WebRequest + 'a> WrappedRequest<'a, R> {
     fn from_err(err: FailParse<R::Error>) -> Self {
         WrappedRequest {
             request: PhantomData,
-            body: Cow::Owned(Default::default()),
+            body: Cow::Owned(NormalizedParameter::default()),
             authorization: None,
             error: Some(err),
             allow_credentials_in_body: false,
         }
     }
 
-    fn parse_header(header: Cow<str>) -> Result<Authorization, Invalid> {
+    fn parse_header(header: impl AsRef<str>) -> Result<Authorization, Invalid> {
+        let header = header.as_ref();
```

Comment:
Can't say I get why this change would be necessary.

Reply:
to be honest i probably refactored that part at 3AM and did random things, looking back this is dumb

will change back